pkgs/development/tools/kafkactl/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+let
+  owner = "deviceinsight";
+in
+  buildGoPackage rec {
```

Comment:
Is there any reason to use `buildGoPackage` instead of `buildGoModule`?

Reply:
Awesome! It was just off my radar that there is a `buildGoModule` now 🎉